android/build.gradle

```diff
@@ -37,9 +36,11 @@ android {
         implementation 'com.segment.analytics.android:analytics:4.10.4'
         implementation 'com.segment.analytics.android.integrations:amplitude:3.1.0'
         implementation 'com.appsflyer:segment-android-integration:6.5.2'
+        implementation 'com.segment.analytics.android.integrations:firebase:2.3.3'
```

Comment:
`aar` package should be preferred to not get `resource not found` errors:

`implementation 'com.segment.analytics.android.integrations:firebase:2.3.3@aar'
`

please see: https://segment.com/docs/connections/destinations/catalog/firebase/#getting-started-on-android

`Note: The Firebase SDK requires android resources which are available on aar packages. Use the aar package when adding the Segment-Firebase SDK.`